magda-migrator-registry-db/sql/V2__Support_multi_tenants.sql

```diff
@@ -0,0 +1,49 @@
+CREATE SEQUENCE IF NOT EXISTS tenants_id_seq;
+
+CREATE TABLE IF NOT EXISTS Tenants (
+    domainName character varying(100) COLLATE pg_catalog."default" NOT NULL,
+    id bigint NOT NULL DEFAULT nextval('tenants_id_seq'::regclass),
+    enabled boolean DEFAULT false,
+    description character varying(1000) COLLATE pg_catalog."default",
+    CONSTRAINT tenants_pkey PRIMARY KEY (domainName),
+    CONSTRAINT tenants_id_key UNIQUE (id),
+    sequence bigserial UNIQUE NOT NULL,
+    lastUpdate bigint REFERENCES Events NOT NULL
+)
+WITH (
+    OIDS = FALSE
+)
+TABLESPACE pg_default;
+
+INSERT INTO Tenants(domainName, id, enabled, description, lastUpdate) VALUES('A built-in id.', 0, true, 'Initial entry used for migrating database from single tenant to multi-tenants.', 1);
+
+ALTER TABLE recordaspects DROP CONSTRAINT recordaspects_recordid_fkey;
```

Comment:
This foreign key constraint needs to be recreated doesn't it?

Reply:
Fixed. Foreign key is a composite key.